### PR TITLE
feat: remove extra retry logic from getpods fn

### PR DESCRIPTION
**Pull request by pritamstyz4ever**
## Context

Verify function has additional retry logic and breaker fn which is not needed as we have the retry queue

## Objective

verify function is called for messages in the retry-queue which already has a retry mechanism built into it, so we don't need this extra call that overwhelms the k8s api.

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by tkyi** [comment]
Do you need to remove retry/circuit breaker logic here as well? https://github.com/screwdriver-cd/executor-k8s/blob/4a57464a3adbd808fab23bf708a12b416fcfea75/index.js#L340

File: index.js [hunk 1]
```diff
@@ -426,18 +426,10 @@ class K8sExecutor extends Executor {
...
-                afterResponse: [this.scheduleStatusRetryStrategy]
-            }
+            https: { rejectUnauthorized: false }
         };
 
-        const resp = await this.breaker.runCommand(statusOptions);
+        const resp = await request(statusOptions);
 
         logger.debug(`Build ${buildId} pod response: ${JSON.stringify(resp.body)}`);
         if (resp.statusCode !== 200) {
```
Comment (tkyi) [inline index.js:435]: Might be better to do try/catch
request package considers anything non-2xx/3xx an error
https://github.com/sindresorhus/got/blob/main/documentation/8-errors.md#httperror

File: index.js [hunk 2]
```diff
@@ -737,19 +729,11 @@ class K8sExecutor extends Executor {
...
-            },
             searchParams: {
                 labelSelector: `sdbuild=${this.prefix}${buildId}`
             }
         };
-        const resp = await this.breaker.runCommand(statusOptions); // list of pods
+        const resp = await request(statusOptions); // list of pods
 
         logger.debug(`Build ${buildId} pod response: ${JSON.stringify(resp.body)}`);
         if (resp.statusCode !== 200) {
```
Comment (tkyi) [inline index.js:739]: same comment here; switch to try/catch

**Review by jithine — APPROVED** [review]

File: index.js [hunk 3]
```diff
@@ -426,35 +426,33 @@ class K8sExecutor extends Executor {
...
-            }
+            https: { rejectUnauthorized: false }
         };
 
-        const resp = await this.breaker.runCommand(statusOptions);
+        try {
+            const resp = await request(statusOptions);
+
+            logger.debug(`Build ${buildId} pod response: ${JSON.stringify(resp.body)}`);
+            if (resp.statusCode !== 200) {
```
Comment (tkyi) [inline index.js:436]: Probably can remove this since the catch block does the same

**Review by tkyi — APPROVED** [review]
Small comments but nonblocking:
- can remove these lines?
https://github.com/screwdriver-cd/executor-k8s/blob/84f68301d18111056590909e0b574aa1b514982e/index.js#L340-L343
https://github.com/screwdriver-cd/executor-k8s/blob/84f68301d18111056590909e0b574aa1b514982e/index.js#L295-L319

File: index.js (hunk 3 above)
Comment (pritamstyz4ever) [reply index.js:436]: but catch block is for error isn't it? If API returns other than 200 status?

**Comment by pritamstyz4ever** [comment]
> Small comments but nonblocking:
> 
> * can remove these lines?
>   https://github.com/screwdriver-cd/executor-k8s/blob/84f68301d18111056590909e0b574aa1b514982e/index.js#L340-L343
>   
>   https://github.com/screwdriver-cd/executor-k8s/blob/84f68301d18111056590909e0b574aa1b514982e/index.js#L295-L319

line 340-343 is needed for updateBuild function, removed 295-319

**Review by jithine — APPROVED** [review]

**Pull request merged**